PEAR/Config.php

```diff
@@ -1022,15 +1022,19 @@ function _readConfigDataFrom($file)
         }
 
         $size = filesize($file);
-        $rt = get_magic_quotes_runtime();
-        set_magic_quotes_runtime(0);
+        if (version_compare(phpversion(), '7.0.0-dev', '<')) {
```

Comment:
I think it would be better to do a `function_exists('set_magic_quotes_runtime')` here, makes the intention clearer.